content/en/docs/Advanced/networking.md

```diff
@@ -27,8 +27,9 @@ stages:
           content: |
             [service_eth0]
             Type = ethernet
-            IPv4 = 10.1.1.1/16/10.1.0.1
```

Comment:
These IP addresses (and the one below) seem to match edgevpn network IPs. Maybe we should have an edgevpn specific example too. Regardless of the IPs, it was the wrong syntax apparently, so :+1: .